<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CORIGINAL</class>
 <widget class="QWidget" name="CORIGINAL">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>810</width>
    <height>566</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="Label_Left">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>0</y>
     <width>95</width>
     <height>65</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Qpbtn_Left">
   <property name="geometry">
    <rect>
     <x>255</x>
     <y>0</y>
     <width>65</width>
     <height>65</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Qpbtn_Right">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>0</y>
     <width>65</width>
     <height>65</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Label_Right">
   <property name="geometry">
    <rect>
     <x>685</x>
     <y>0</y>
     <width>125</width>
     <height>65</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="QpbtnLead">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>0</y>
     <width>80</width>
     <height>65</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="QpbtnBack">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>80</width>
     <height>65</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="QpbtnStretching">
   <property name="geometry">
    <rect>
     <x>128</x>
     <y>340</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>65</y>
     <width>821</width>
     <height>503</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <property name="spacing">
     <number>0</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLineEdit" name="SearchData">
          <property name="minimumSize">
           <size>
            <width>117</width>
            <height>34</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>117</width>
            <height>34</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="QpbtnSearch">
          <property name="minimumSize">
           <size>
            <width>34</width>
            <height>34</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>34</width>
            <height>34</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTreeWidget" name="QDepartment">
        <property name="minimumSize">
         <size>
          <width>161</width>
          <height>456</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>161</width>
          <height>456</height>
         </size>
        </property>
        <column>
         <property name="text">
          <string notr="true">1</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableWidget" name="QOriginalAttendance">
      <property name="minimumSize">
       <size>
        <width>650</width>
        <height>501</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="Button_Year">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>0</y>
     <width>150</width>
     <height>65</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_Month">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>0</y>
     <width>150</width>
     <height>65</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <zorder>Label_Left</zorder>
  <zorder>Qpbtn_Left</zorder>
  <zorder>Qpbtn_Right</zorder>
  <zorder>Label_Right</zorder>
  <zorder>QpbtnLead</zorder>
  <zorder>QpbtnBack</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>QpbtnStretching</zorder>
  <zorder>Button_Year</zorder>
  <zorder>Button_Month</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
